doc/user_guide/editions.md

```diff
@@ -1,56 +1,23 @@
-
-## Selecting your Edition
-
-Please see common [system requirements](user_guide.md#system-requirements) for all editions.
+Please see common [system requirements](system-requirements.md) for all editions and select your favorite edition depending on your operating system, infrastructure, administrative permissions, and technical skills.
 
 Recommendations
-* If you have an AWS account then Exasol recommends to use the AMI image in an AWS EC2 instance.
+* If you have an AWS account then Exasol recommends to use the AMI Edition running the AMI image in an AWS EC2 instance.
 * In case a Docker client is available on your system then probably the Docker Edition is the best choice.
-* When you want to use one of the VM images then select an appropriate format depending on the Hypervisor software available on your system:
-
-### AMI Edition
-
-The ID of the AMI (Amazon Machine Image) is mentioned in the [release notes](https://github.com/exasol/ai-lab/releases/latest) and can be used to start an EC2-instance in your AWS account.
-
-The naming scheme is: "_Exasol-AI-Lab-${VERSION}_", e.g. "_Exasol-AI-Lab-5.0.0_"
+* When you want to use the VM Edition then select an appropriate VM image format depending on the Hypervisor software available on your system.
 
-See also [User Guide for AI-Lab AMI Edition](ami_usage.md).
 
-### Virtual Machine Edition
+| Description                              | Format(s)                                                                |
+|------------------------------------------|--------------------------------------------------------------------------|
+| [AMI Edition](ami-usage.md)              | Amazon Machine Image (AMI)                                               |
+| [Docker Edition](docker/docker-usage.md) | Docker Image                                                             |
+| [Virtual Machine Edition](vm-usage.md)   | VMware Virtual Machine Disk (VMDK), Virtual Hard Disk by Microsoft (VHD) |
 
-The following table shows which Hypervisor supports which image formats:
-
-| Hypervisor          | VMDK | VHD |
-|---------------------|------|-----|
-| VMWare work station | yes  | ?   |
-| Virtual box         | yes  | yes |
-| Hyper-V             | ?    | yes |
-| WSL 2               | ?    | ?   |
-
-Notes for Hypervisors based on KVM/[QEMU](https://en.wikipedia.org/wiki/QEMU):
-* There multiple alternative UIs, see https://en.wikipedia.org/wiki/Libvirt#User_Interfaces.
-* File in format VHD or VMDK needs to be converted first, see
-  * https://docs.openstack.org/image-guide/convert-images.html
-   * https://linux.die.net/man/1/qemu-img
-
-See also [User Guide for AI-Lab VM Edition](vm_usage.md).
-
-### Docker Edition
-
-The Docker image is published to DockerHub at https://hub.docker.com/r/exasol/ai-lab.
+Each of the editions is associated with an _image_ in a specific format which
+* Is linked in the [release notes](https://github.com/exasol/ai-lab/releases/latest) for download
```

Comment:
It doesn't sound right to me.
Maybe "Available for download here" or "(see [release notes])" or "(go to the [release notes])".